app/models/blog.rb

```diff
@@ -14,7 +14,7 @@ class Blog < ApplicationRecord
   scope :default_order, -> { order(id: :desc) }
 
   scope :viewable, lambda { |user|
-    where('user_id = ?', user&.id).or(published)
+    where('blogs.user_id = ?', user&.id).or(published)
```

Comment:
```suggestion
    where(user: user).or(published)
```

でも同じことが実現できそうです。
ログに出力されるSQLをチェックしてみてください〜。

Reply:
ログを確認したところ、同じSQLが発行されることが確認できました。
確かに、`where(user: user).or(published)`の方がコードが読みやすそうですね。

`where(user: user).or(published)`に修正をしました✍️